utils/estimated_histogram.hh

```diff
@@ -360,8 +360,6 @@ public:
         s += "]";
         return s;
     }
-
-    }
```

Comment:
There is another, more efficient way to do this, and that is by using pseudo floating point. For example we can have 4 significant bits and then a value of 195 = 0xc3 = 0y1100'0011 = (128) * (1 + (0y100'0011 / 128)) =~ 128 * (1 + 0y100 / 8) = (1<<7)*(1 + 4/8)

The bucket number is therefore 7*8+4 = 60. To recover the value again divide by 8 (we get 7 with a reminder of 4), and write (1<<7) + (4 << (7 - 3)) = 128 + 64 = 192.

Reply:
I'll have a look

Reply:
I've openned https://github.com/scylladb/scylla/issues/5815